client/src/InfoList.jsx

```diff
@@ -5,7 +5,7 @@ import { OpeningHours } from './OpeningHours.jsx';
 var InfoList = (props) => {
```

Comment:
This could potentially be a great opportunity for destructuring to make this code even cleaner (and use airbnb style guide): something like
```
var InfoList = ({ restaurant: { open_now, periods, weekday_text, url } }) => {

...

    data: { open_now, periods, weekday_text },

...

    text: (new URL(url)).hostname,

...etc
```

not totally necessary but just throwing it out there! it can make for pretty clean lines of code when dealing with props :)

Reply:
Yeah, I thought about doing that and it is a great idea. I was just trying to do the bare minimum changes I could on the front end inherited code and sort of felt that if I got started destructuring I would start to want to implement destructured props throughout the front end, so I decided to just leave it. Thanks for flagging this though, definitely will do it if I end up doing more work on the front end.

Reply:
That makes total sense -- probably not worth the investment here since we're trying to touch the front end as little as possible 👍